docs/installation.md

```diff
@@ -35,9 +35,18 @@ Create a file called `${CONTEXT}.xml` in
 necessary, and populate it with the following information:
 
     <Context>
-       <Parameter name="runscanner.configFile" value="/etc/runscanner.json" override="false"/>
+        <Parameter name="runscanner.configFile" value="/etc/runscanner.json" override="false"/>
+        <Parameter name="swagger.baseUrl" value="http://runscanner:8080"/>
     </Context>
 
+Base URL can usually be determined automatically, but may need manual setting if behind a reverse
+proxy, depending on its configuration:
+`swagger.baseUrl=http://runscanner:8080`
```

Comment:
- Clarify that `swagger.baseUrl` is optional by removing it from the base example above and describe here how to add it if needed. Correct the format below.